docs/_indicators/Keltner.md

```diff
@@ -70,4 +70,22 @@ See [Utilities and helpers]({{site.baseurl}}/utilities#utilities-for-indicator-r
 
 ## Chaining
 
-This indicator is not chain-enabled and must be generated from `quotes`.  It **cannot** be used for further processing by other chain-enabled indicators.
+Results can be further processed on `Centerline` with other [chained indicators]({{site.baseurl}}/guide/#chaining).
```

Comment:
Chaining section of guide needs to be restored, somewhere it must have been accidentally removed.